.github/workflows/pmm-ui-tests-matrix-upgrade.yml

```diff
@@ -51,3 +51,19 @@ jobs:
         pmm_qa_branch: ${{ inputs.pmm_qa_branch || 'main' }}
         testing_repository: ${{ inputs.testing_repository }}
         experimental_repository: ${{ inputs.experimental_repository }}
+
+   general:
+     name: General
+     uses: ./.github/workflows/pmm-ui-tests-upgrade.yml
+     secrets: inherit
+     with:
+       pmm_ui_tests_branch: ${{ inputs.pmm_ui_tests_branch || 'main' }}
+       pmm_pre_upgrade_flag: '@pre-upgrade'
```

Comment:
We talked about using features flags for example @backup-pre-upgrade @backup-post-upgrade

Reply:
Yes and these tests are exceptions which have to be run separately from it's feature "BM", but default settings check does not deserve a dedicated feature at least for now. Yet we will have test which will be classified as general checks or will cross multiple features and will require a dedicated group. So I suppose here is our start with "General" group and "RBAC" already have own group.

Reply:
moved to "configuration" feature